server/index.js

```diff
@@ -0,0 +1,120 @@
+// server/index.js
+
+import "dotenv/config";
+
+import child_process from "child_process";
+import cors from "cors";
+import express from "express";
+
+const PORT = 8000;
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.post("/start-kraken/", (req, res) => {
+  const passwd = req.headers?.authorization?.split(" ")[1];
+  const scenario = req.body.params.scenarioChecked;
+  let command = "";
+  switch (scenario) {
+    case "pod-scenarios":
+      command = `echo '${passwd}' | sudo -S podman run --env NAMESPACE=${req.body.params.namespace} --env NAME_PATTERN=${req.body.params.name_pattern} --env POD_LABEL=${req.body.params.pod_label} --env DISRUPTION_COUNT=${req.body.params.disruption_count}  --env KILL_TIMEOUT=${req.body.params.kill_timeout} --env WAIT_TIMEOUT=${req.body.params.wait_timeout} --env EXPECTED_POD_COUNT=${req.body.params.expected_pod_count} --name=ui --net=host --env-host -v ${req.body.params.kubeconfigPath}:/root/.kube/config:Z -d quay.io/redhat-chaos/krkn-hub:pod-scenarios`;
```

Comment:
Maybe we should add checks to make sure podman is installed? Thoughts?

Reply:
Okay. Will work on it

Reply:
Included the check for podman status